src/components/BooksList.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
```

Comment:
- [ ] `BookList` component should be placed in containers folder instead of components folder. Anything connects to redux store should be put inside containers folder. This pattern is for organizing files and making the distinguish between presentational and container components.

Please check [here](https://reactjs.org/docs/typechecking-with-proptypes.html) for more about this